bootcamp-milestone-2/src/app/components/navbar.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import style from "./navbar.module.css";
+import Link from "next/link";
+
+export default function Navbar() {
+  return (
+    // replace everything in between the <header> & <header /> tags
+    // with your navbar code from your earlier milestones
+    <header className={style.navbar} >
+        <nav className="navbar"> 
+        <h1 className="logo"> <a href="index.html"> Elaina Lyons </a> </h1>
```

Comment:
When I click on your name, it takes me to "index.html" which yields a 404 not found error. Consider how you might want to alter this- a couple ideas could be to remove this link altogether, or maybe route to the home page?